dev/VSIX/ProjectTemplates/Desktop/CppWinRT/PackagedApp/BlankApp/WinUI.Desktop.CppWinRT.BlankApp.vstemplate

```diff
@@ -21,7 +21,7 @@
     <LanguageTag>cpp</LanguageTag>
     <LanguageTag>XAML</LanguageTag>
     <PlatformTag>windows</PlatformTag>
-    <PlatformTag>Project Reunion</PlatformTag>
+    <PlatformTag>Windows App SDK</PlatformTag>
```

Comment:
>Windows App SDK [](http://example.com/codeflow?start=17&length=15)

wondering whether WindowsAppSDK would be better?

Reply:
Good question. It had a space so I kept the same style notation.

I don't know if there's technical implications (either way). Who's driving the template work? @AdamBraden do you know who should follow up?